bors.toml

```diff
@@ -4,4 +4,7 @@ status = [
     "build_test (x86_64-apple-darwin)",
     "docker_tests (happy_path)",
     "docker_tests (happy_path_restart_bob_before_comm)",
+    "docker_tests (bob_refunds_using_cancel_and_refund_command.rs)",
```

Comment:
Don't think you want the `.rs` extension here.

Reply:
Yes, typo already corrected.